### ISSUE TITLE
Are there some FOSS VULCAN server? Can it be integrated with Moodle?

### ISSUE THREAD

**Issue by timofonic** [luk1337/TimeTable2#2]
Hello.

I see the VULCAN system quite interesting. I wonder if are there some open standard based on it or equivalent. Are there?

I'm trying to convince my vocational school to update their Moodle installation, because the November version seems quite interesting. But I miss some way to help me not forget classes and prepare to move to next one without getting distracted.

I found little information about VULCAN, specially the server part. I just found Vulcan.edu.pl and it seems to be a proprietary service, unfortunately.

I'm time blind and work better with digital than paper, so anything that can help me at managing tasks would be a great help.

Kind regards.

**Comment by luk1337** [comment]
No. It's proprietary and costs a fortune lol. I dunno why schools are using it here.

**Comment by luk1337** [comment]
Anyway I could probably add moodle support but I need to have a live demo page.
